desertbot/modules/commands/Comic.py

```diff
@@ -0,0 +1,235 @@
+"""
+Created on Apr 18, 2019
+
+@author: lunik1
+"""
+import base64
+import glob
+import json
+from io import BytesIO
+from random import shuffle
+
+import requests
+from PIL import Image, ImageDraw, ImageFont
+from twisted.plugin import IPlugin
+from zope.interface import implementer
+
+from desertbot.message import IRCMessage
+from desertbot.moduleinterface import IModule
+from desertbot.modules.commandinterface import BotCommand
+from desertbot.response import IRCResponse, ResponseType
+
+
+@implementer(IPlugin, IModule)
+class Comic(BotCommand):
+    # just hardcode a limit on max messages for now, should limit comic size pretty effecively?
+    messageLimit = 8
+
+    def actions(self):
+        # define additional actions so all PRIVMSGs and ACTIONs in the channel get handled by storeMessage
+        return super(Comic, self).actions() + [("message-channel", 1, self.storeMessage),
+                                               ("action-channel", 1, self.storeMessage)]
+
+    def onLoad(self) -> None:
+        # messages need to be stored for comics to work, store in a list as (nick, message)
+        self.messageStore = []
```

Comment:
this should be a dict so that lists are stored per-channel

Reply:
fixed